.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+## Contribute to Magento Glossary
+
+Share your knowledge with the community by contributing to Magento Glossary, part of Magento DevDocs!
+You can contribute by creating an issue or pull request (PR) on our [Glossary](https://github.com/magento/glossary) GitHub repository.
+We welcome all types of contributions; from minor typo fixes to new terms.
+
+Magento's tech writer team and [Community Maintainers](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributors.html#/community-maintainers) review issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible.
+Working through the backlog takes time, though, so we appreciate your patience.
+
+## Rewards for contributions
+
+DevDocs works with Magento Community Engineering teams and projects.
+As you contribute PRs, you gain [Contribution Points](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#points). We track [Contributors](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributors.html#/individual-contributors) and [Maintainers](https://devdocs.magento.com/guides/v2.3/contributor-guide/maintainers.html) with Community Engineering.
+
+## Get started
+
+![Get started workflow](https://devdocs.magento.com/common/images/contribute-prerequisites.png)
+
+1. Make sure you have a [GitHub account](https://github.com/signup/free) with [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#two-factor)(2FA) to your account. Partners are required to add 2FA protection when contributing to Magento repositories.
```

Comment:
This is awkward. Consider using recent revisions to devdocs contribution guidelines: https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md#get-started

Reply:
should we just update this file to point to the DevDocs contribution guidelines.
that way we don't have duplicate content that need constant updating?

Reply:
Yes! That's a great idea.